service/init.go

```diff
@@ -435,16 +435,14 @@ func (udm *UDM) registerNF() {
 		if err != nil {
 			logger.InitLog.Errorln(err.Error())
 		} else {
-			var newNrfUri string
 			var err1 error
 			var prof models.NfProfile
-			prof, newNrfUri, self.NfId, err1 = consumer.SendRegisterNFInstance(self.NrfUri, self.NfId, proflie)
+			prof, _, self.NfId, err1 = consumer.SendRegisterNFInstance(self.NrfUri, self.NfId, proflie)
```

Comment:
we can also correct typo proflie -> profile

Reply:
Done, nice catch!